field.scss

```diff
@@ -18,3 +18,30 @@
 	}
 
 }
+
+@mixin vui-field-label($inline: false) {
+
+	color: $vui-color-ferrite;
+	cursor: default;
+	display: block;
+	font-size: 0.7rem;
```

Comment:
Perhaps make use of vui-typography mixin [vui-typography-small-text](https://github.com/Brightspace/valence-ui-typography/blob/master/typography.scss#L97) - I suspect the intent was to use that style for labels, but I could be wrong.  Besides eliminating the need to specify color and size, it also specifies character spacing which is not included here.  The mixin allows overriding the margin, but you'd have to add line-height as a param if you were to use it.